raiden-dapp/src/utils/balance-utils.ts

```diff
@@ -20,6 +24,18 @@ export class BalanceUtils {
   }
 
   static parse(deposit: string, decimals: number) {
-    return parseUnits(deposit, decimals);
+    return parseUnits(
+      deposit.endsWith('.')
+        ? deposit.substring(0, deposit.length - 1)
+        : deposit,
+      decimals
+    );
+  }
+}
+
+export function getDecimals(decimals?: number) {
+  if (decimals === undefined) {
```

Comment:
No need to change it, but this could be `decimals ?? 18`, and if it'd be this simple, maybe there'd be no need for a function

Reply:
That is a nice suggestion, I didn't think of that to be honest